diff --git a/src/rosunit/xmlrunner.py b/src/rosunit/xmlrunner.py
--- a/src/rosunit/xmlrunner.py
+++ b/src/rosunit/xmlrunner.py
@@ -199,7 +199,7 @@
         output and standard error streams must be passed in.a
 
         """
-        stream.write(ET.tostring(self.xml(time_taken, out, err).getroot(), encoding='utf-8', method='xml'))
+        stream.write(ET.tostring(self.xml(time_taken, out, err).getroot(), encoding='unicode', method='xml'))
 
     def print_report_text(self, stream, time_taken, out, err):
         """Prints the text report to the supplied stream.
