diff --git a/osdk-core/platform/linux/inc/linux_serial_device.hpp b/osdk-core/platform/linux/inc/linux_serial_device.hpp
index 5e6d034..bf75eb4 100644
--- a/osdk-core/platform/linux/inc/linux_serial_device.hpp
+++ b/osdk-core/platform/linux/inc/linux_serial_device.hpp
@@ -40,6 +40,7 @@
 #include <fcntl.h>
 #include <termios.h>
 #include <unistd.h>
+#include <sys/select.h>
 
 #include "dji_hard_driver.hpp"
 
