diff --git a/dji_sdk/CMakeLists.txt b/dji_sdk/CMakeLists.txt
index 9d92156..dfdc657 100644
--- a/dji_sdk/CMakeLists.txt
+++ b/dji_sdk/CMakeLists.txt
@@ -22,7 +22,7 @@ set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
 ## Uncomment this if the package has a setup.py. This macro ensures
 ## modules and global scripts declared therein get installed
 ## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
-catkin_python_setup()
+# catkin_python_setup()
 
 ################################################
 ## Declare ROS messages, services and actions ##
